PowerPointLabs/PowerPointLabs/TextCollection.cs

```diff
@@ -98,6 +98,8 @@ internal class TextCollection
         public const string EffectsLabMagnifyGlassSupertip = @"Magnify a small area or detail on the slide.";
         public const string EffectsLabBlurRemainderSupertip = @"Draw attention to an area of the slide by blurring everything else.";
         public const string EffectsLabColorizeRemainderSupertip = @"Recolor an area of a slide to attract attention to it.";
+        public const string EffectsLabBlurBackgroundSupertip = @"Blur everything in the slide except for this shape.";

```

Comment:
The feature works for multiple selections (I think). So the wording needs to change. `... except for selected shapes`?